cpp/include/cuspatial/experimental/detail/ranges/enumerate_range.cuh

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @brief Given Any Range, zip the iterator with the index of the element.
```

Comment:
Does this standalone brief work without @file?

Reply:
I don't think so, but at the same time, I'm not sure if we should expose this to public API? See I left it in `detail`.

Reply:
I think delete this standalone comment block. The one on the function should be sufficient.

Reply:
Removed